README.md

```diff
@@ -1884,15 +1884,20 @@ Now that we have the result of the fetch, let's now map each item, and then extr
 ```js
 dataPoint.addEntities({
   'request:getOrgRepositories': {
-    url: 'https://api.github.com/orgs/nodejs/repos'
+    url: 'https://api.github.com/orgs/nodejs/repos',
+    options: {
```

Comment:
not sure if it would be better to omit the options object! It does take up alot of room, however if someone is copying and pasting to see the code work for themselves (i.e. me an hour ago haha), they'll get an error without it

Reply:
I think its totally worth it, tho we should move this example to use swapi, for now this is great, thanks!